syz-fuzzer/proc.go

```diff
@@ -132,6 +133,14 @@ func (proc *Proc) triageInput(item *WorkTriage) {
 			}
 			continue
 		}
+		inf := &info.Extra
+		if item.call != -1 {
```

Comment:
I think this should be a part of getSignalAndCover, it already contains this if.